src/main/kotlin/no/nav/syfo/client/narmesteleder/NarmesteLederClient.kt

```diff
@@ -22,11 +22,13 @@ class NarmesteLederClient(
     private val narmestelederClientId: String,
     private val azureAdV2Client: AzureAdV2Client
 ) {
-    private val sykmeldtAktivNarmesteLederPath = "$narmesteLederBaseUrl$NARMESTELEDER_CURRENT_PATH?orgnummer="
+    private val sykmeldtAktivNarmesteLederPath = "$narmesteLederBaseUrl$CURRENT_NARMESTELEDER_PATH?orgnummer="
+    private val sykmeldtNarmesteLederePath = "$narmesteLederBaseUrl$NARMESTELEDERE_PATH?utvidet=ja"
+    private val ansatteNarmesteLederPath= "$narmesteLederBaseUrl$CURRENT_ANSATTE_PATH"
 
     private val httpClient = httpClientDefault()
 
-    suspend fun activeLeader(
+    suspend fun activeLederrelasjon(
```

Comment:
Hvis denne skal hete "lederrelasjon" burde ikke de andre også hete noe med "Relasjon"? Alle returnerer jo (en liste av) samme objekt?

Reply:
Hvis jeg husker rett, så er det fordi NL API-et wrapper respons av en NL med `{ narmesteLederRelasjon: { ... }`, men det for endepunktet hvor man får flere NL er de ikke wrappet med narmesteLederRelasjon. Hvis det gir mening? 😅

Reply:
Ja, skjønner! 👍 Men jeg tenker kanskje denne metoden ikke burde hete relasjon da, siden den "unwrapper" og returnerer en `NarmesteLederDTO?`, ikke en Relasjon?

(For meg virker det litt rart at man sier `return narmesteLederRelasjon.narmesteLederRelasjon` og så gir det en NarmestelederDTO, men det har jo ikke noe med oppgaven deres å gjøre 🤷‍♂️ )